src/shell/command_helper.h

```diff
@@ -630,6 +637,21 @@ update_app_pegasus_perf_counter(row_data &row, const std::string &counter_name,
         row.rdb_estimate_num_keys += value;
     else if (counter_name == "backup_request_qps")
         row.backup_request_qps += value;
+    else if (counter_name == "get_bytes")
```

Comment:
What will shell command change? Could you please add some explanations to me?

Reply:
Haven't support shell.

Reply:
Ok, could you please explain why you update those perf-counters in function `update_app_pegasus_perf_counter`?

Reply:
Oh, I only add the statistic as previous for `shell`.

Reply:
I suggest that if you add those counters in this function, you should update related shell command, otherwise, I suggest you remove those useless code, and add them if needed.